common/app/conf/switches.scala

```diff
@@ -807,6 +807,15 @@ object Switches {
     exposeClientSide = true
   )
 
+  val RugbyWorldCupMatchStatsSwitch = Switch(
+    "Feature",
+    "rugby-world-cup-match-stats",
+    "If this switch is on rugby world cup stats will be loaded in to rugby match reports and liveblogs",
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Use the same sell by date as `RugbyWorldCupswitch` which is `new LocalDate(2015, 11, 6)`